### ISSUE TITLE
No contrast

### ISSUE THREAD

**Issue by Chrisantho** [QuantumEmpress/Fitness#3]
The text "Subscribe Our Newsletter and Get 20 Days Free Trial"  close to the bottom of the page is not visible due to color contrast.
You can try changing the text to white.
